tcf_website/templates/base/index.html

```diff
@@ -23,7 +39,7 @@
     })(window,document,'script','dataLayer','GTM-54T5TXX');</script>
     <!-- End Google Tag Manager -->
     
-    <script src="https://kit.fontawesome.com/7a4b03e605.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/ed3998fccc.js" crossorigin="anonymous"></script>
```

Comment:
why did you update this lol

Reply:
Lol I do not remember the context, I might have got it from some other branch??? I think I used master or smt instead of dev

Sry for that 😭